### ISSUE TITLE
The accuracy is so low

### ISSUE THREAD

**Issue by catchinmyeyes** [wsyCUHK/WSYCUHK_FDIA#5]
The accuracy is 0.0910 only.

**Comment by itxsc** [comment]
> The accuracy is 0.0910 only.

Can you share the method of training data generation? This is very important to me. My email is itxsc@itxsc@hrbeu.edu.cn. Regarding this article, I look forward to further communication with you.

**Comment by itxsc** [comment]
Can you share the method of training data generation? This is very important to me. My email is itxsc@hrbeu.edu.cn. Regarding this article, I look forward to further communication with you.

**Comment by ShuchunXu** [comment]
> The accuracy is 0.0910 only.

hi, catchinmyeyes,
 Can you share the method of training data generation? This is very important to me. My email is [itxsc@hrbeu.edu.cn](mailto:itxsc@hrbeu.edu.cn). Regarding this article, I look forward to further communication with you.
@catchinmyeyes @wsyCUHK

**Comment by boom623** [comment]
Hello, Doctor. I recently ran your code on GitHub and modified the split of the validation set and the training set to 7:3. The final accuracy was only 8.99.
